bindings/nodejs/index.d.ts

```diff
@@ -6,14 +6,17 @@
 export class OperatorFactory {
   static memory(): Operator
 }
-export class ObjectMeta {
+export class Operator {
+  object(path: string): object
+}
+export class ObjectMetadata {
   location: string
   lastModified: number
   size: number
 }
-export class Operator {
-  meta(path: string): Promise<ObjectMeta>
-  write(path: string, content: Array<number>): Promise<void>
-  read(path: string): Promise<Array<number>>
-  delete(path: string): Promise<void>
+export class Object {
+  meta(): Promise<ObjectMetadata>
+  write(content: Array<number>): Promise<void>
```

Comment:
I guess nodejs has native thing like `bytes`? `Array<number>` seems need an extra alloction.

Reply:
We can use `Buffer` as argument and result.

Reply:
Great!